edx_django_utils/monitoring/internal/utils.py

```diff
@@ -19,9 +19,23 @@
 """
 from contextlib import ExitStack, contextmanager
 
+from django.conf import settings
+
 from .backends import configured_backends
 from .middleware import CachedCustomMonitoringMiddleware
 
+DJANGO_MANAGEMENT_MONITORING_OPERATION_NAME = getattr(
```

Comment:
Can you annotate the settings. Follow an example like: https://github.com/openedx/edx-platform/blob/45f44c3965498fef80dcde676f2969324820a590/openedx/core/djangoapps/oauth_dispatch/jwt.py#L125-L130